### PR TITLE
Increase ssh tunnel's kChunkSize for faster reads

### PR DESCRIPTION
**Pull request by pierricgimmig**
This greatly reduces the hangs we are sometimes seeing on capture
stop. The real cause is the fact that the tunnel is updated on the ui
thread, which shouldn't be the case as it can being drastically delayed 
by a long render frame.

See b/172686811

### CODE REVIEW

File: OrbitSshQt/Tunnel.cpp [hunk 1]
```diff
@@ -180,6 +180,7 @@ outcome::result<void> Tunnel::readFromChannel() {
     if (bytes_written == -1) {
       SetError(Error::kLocalSocketClosed);
     } else {
+      // TODO: avoid extra copy b/172686811.
```
Comment (dpallotti) [inline OrbitSshQt/Tunnel.cpp:183]:
```suggestion
      // TODO(b/172686811): avoid extra copy.
```

**Review by dpallotti — APPROVED** [review]

**Review by dimitry- — APPROVED** [review]

**Comment by ronaldfw** [comment]
Should this actually go into release/1.55 or only master at this point?

**Comment by dpallotti** [comment]
> Should this actually go into release/1.55 or only master at this point?

Ah right, this is actually targeting the release branch...

**Comment by dimitry-** [comment]
Right - this should go to master.

**Comment by ronaldfw** [comment]
Per discussion with @pierricgimmig, this should now actually go into release/1.55.

**Pull request merged**